tools/ProgressOnderwijsUtilsBenchmarks/NullabilityBenchmark.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Toolchains.InProcess.NoEmit;
+using Perfolizer.Mathematics.OutlierDetection;
+using ProgressOnderwijsUtils.Tests;
+using Xunit.Sdk;
+
+namespace ProgressOnderwijsUtilsBenchmarks;
+#nullable enable
```

Comment:
```suggestion
```

even gedubbelchecked, maar dit hebben we idd voor alle projects geregeld, en namelijk hier: https://github.com/progressonderwijs/ProgressOnderwijsUtils/blob/master/src/Common.props#L16

Bijzonder trouwens dat er geen warning is voor een redundante nullability-checking pragma die gelijk is aan de default; ach ja...